controllers/__init__.py

```diff
@@ -316,3 +317,17 @@ def delete_link(self, link_id: str) -> Optional[dict]:
         return self.db.links.find_one_and_delete(
             {"_id": link_id}
         )
+
+    def delete_switch_data(
+        self,
+        switch_dpid: str
+    ) -> tuple[Optional[dict], int]:
+        """Delete a switch related data in database."""
+        regex = re.compile(rf'^{switch_dpid}:\d+$')
```

Comment:
Side note: the regex looks good and nicely done that you anchored it, I expect it to perform well since `_id` is indexed too, but just in case when you're locally exploring this, please keep an eye on APM or increase the profiling level of the collection and observe if under expected cases it's only taking a few ms.

Reply:
Fixing `coloring` and will post the APM results later.

Reply:
Here are the results:

From Mongo:
![3](https://github.com/kytos-ng/topology/assets/55767214/1a16fa00-48c6-4579-8b7d-170c3227768e)

From Kytos:
![4](https://github.com/kytos-ng/topology/assets/55767214/dc58363c-40d4-4720-a835-34cbe0a3ca79)